ksonnet-gen/ksonnet/util.go

```diff
@@ -26,6 +26,8 @@ var specialProperties = map[kubespec.PropertyName]kubespec.PropertyName{
 	"kind":       "kind",
 }
 
+var specialPropertiesList = []string{"apiVersion", "kind"}
```

Comment:
I think `specialPropertiesList := []string{"apiVersion", "kind"}` is more go-like, but if this is the style of this codebase, then by all means.